### PR TITLE
allow learning rate to be configurable

### PR DESCRIPTION
**Pull request by markgllin**
allow learning rate to be configurable. Default was 0.001 [as set in pytorch](https://pytorch.org/docs/stable/_modules/torch/optim/adam.html#Adam)

### CODE REVIEW

**Comment by oalieno** [comment]
Looks good.
